src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -296,6 +298,8 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
     }
 
     private renderFontSection() {
+        const defaultFontSize = SettingsStore.getValue(UIFeature.DefaultFontSize) || 10;
+        if ((!SettingsStore.getValue(UIFeature.ChangeFont))) return null;
```

Comment:
Please remove additional brackets ... I have done this in a commit please pull

Reply:
Thanks for doing that one.